### DISCUSSION TITLE
How to design a flow with two projects

### DISCUSSION THREAD

**Discussion by YishuiLi** [PrefectHQ/prefect#3872] [General]
There're two python projects and each project defined some tasks. How to design a flow with the tasks in these two projects.
For example:
project A: task1, task2, task3
project B: task4, task5, task6

I want to design a flow: task1->task2->task3->task4->task5->task6. I don't want to merge these two project. How to design this flow.

**Comment by zanieb** [comment]
Hi! You can just import the tasks from the python module of each project into the file you're writing the flow in.

Either add your project to the PYTHONPATH so it's importable or create a `setup.py` and install the project
https://docs.python.org/3/using/cmdline.html#envvar-PYTHONPATH

**Reply by YishuiLi** [reply]
It worked! Thank you!
And I got another question. If these two projects run on different machines.  Like two django/flask projects run on different machines, is there any way I can design this flow?

**Reply by zanieb** [reply]
It depends if you're talking about a single flow or not. Flows can be labeled to run on different machines https://docs.prefect.io/orchestration/agents/overview.html#labels so you could have each flow run where the project is available. You cannot easily schedule the tasks of a single flow to run on different machines though and I wouldn't recommend having a flow with split dependencies like that. You could use the `StartFlowRun` task within a flow to run two other flows on these separate environments, but that gets complicated.

**Reply by YishuiLi** [reply]
This is very helpful! Thank you!
